client/src/views/Environment.tsx

```diff
@@ -19,51 +20,69 @@ const ENDPOINTS_POLLING_INTERVAL = 5000;
 
 function Environment() {
   const theme = useTheme();
-  const { output, environment, stopEnvironment, relaunchEnvironment } = useOkteto();
+  const { output, environment, status, stopEnvironment, currentContext, relaunchEnvironment } = useOkteto();
   const [endpoints, setEndpoints] = useState<Array<string>>([]);
+  const prevStatusRef = useRef<OktetoStatus | null>(null);
+
 
   const handleOpenEnvironment = () => {
     if (environment) {
       window.ddClient.host.openExternal(environment.link);
     }
   };
 
+  useEffect(() => {
+    if ( prevStatusRef.current && prevStatusRef.current === 'activating' && status === 'synchronizing')
```

Comment:
This should be shown when the status is `ready`, not syncronizing nor activating.

@pchico83 I've been testing this and it seems that the Okteto CLI "status" command is never returning the `ready` status, it always ends in `syncronizing` for some reason. Should I escalate this to the CLI issues?

Reply:
@ this is the only thing pending?

Reply:
I think so, we need to make sure the `status` command is working properly before releasing this. @pchico83 could you take a look?

Reply:
@pchico83 any update one this?

Reply:
@aryabharat not yet, I will work on it later this week, probably on Thursday

Reply:
@pchico83 any updates?

Reply:
yes, I thought I sent you an update. we made a new releases that fixes the `status` command. you need to upgrade your okteto extension to get it

Reply:
@aryabharat The condition here is still considering "synchronizing" to show the success message and that should be changed now that the CLI is returning the right status.

Reply:
Hi @aryabharat, any progress in this? The PR is almost done, we just need the right status check, and it'll be good to go!

Reply:
Hi @rlamana, was not available last few weeks,  will try to close this PR by this week.
Thanks for the patience.

Reply:
No problem! Thank you for your contribution!

Reply:
Hi @rlamana! while local testing iam not able to get the status.
can you help here.

![image](https://user-images.githubusercontent.com/22941703/208098366-a123b4e9-18e0-4802-8d0d-1d85c9494721.png)

Reply:
Hi @aryabharat! Did you upgrade your okteto extension to get @pchico83's changes in the latest release?

Reply:
Hi @rlamana !
I have update the version and fixed the condition.
Please check.
Let me know if anything else has to be done.

Reply:
Hi @aryabharat ! Thank you! I'll need to test this a bit before merging

Reply:
@rlamana sure. Let me know if any changes are required.